doc/source/configuration/ci-cd.rst

```diff
@@ -95,6 +95,11 @@ Runner Deployment
       - openstack
       - "{{ kayobe_environment | default(omit) }}"
 
+    github_registry:
```

Comment:
This wouldn't work in a non-shared multiple environment setup, would it?

Reply:
Yes if the user had provided the above dict for `github_registry` it would be a single registry for the multiple environments however `github_registry['share'] = true` is missing which would cause build image to build multiple images to the same registry. More importantly it wouldn't work if the registry was different between the environments as the variables are without their environment prefix.